airflow/dag_processing/manager.py

```diff
@@ -298,74 +301,58 @@ class DagFileProcessorManager(LoggingMixin):
     :param signal_conn: connection to communicate signal with processor agent.
     """
 
-    def __init__(
-        self,
-        dag_directory: os.PathLike[str],
-        max_runs: int,
-        processor_timeout: timedelta,
-        signal_conn: MultiprocessingConnection | None = None,
-    ):
-        super().__init__()
-        # known files; this will be updated every `dag_dir_list_interval` and stuff added/removed accordingly
-        self._file_paths: list[str] = []
-        self._file_path_queue: deque[str] = deque()
-        self._max_runs = max_runs
-        # signal_conn is None for dag_processor_standalone mode.
-        self._direct_scheduler_conn = signal_conn
-        self._parsing_start_time: float | None = None
-        self._dag_directory = dag_directory
-        # Set the signal conn in to non-blocking mode, so that attempting to
-        # send when the buffer is full errors, rather than hangs for-ever
-        # attempting to send (this is to avoid deadlocks!)
-        if self._direct_scheduler_conn:
+    _dag_directory: os.PathLike[str]
+    max_runs: int
+    processor_timeout: float = attrs.field(factory=_config_int_factory("core", "dag_file_processor_timeout"))
+    selector: selectors.BaseSelector = attrs.field(factory=selectors.DefaultSelector)
+    _direct_scheduler_conn: MultiprocessingConnection | None = attrs.field(alias="signal_conn", default=None)
```

Comment:
Why do we need alias? Should we instead just use `signal_conn` as name?

Reply:
I am guessing you are using it so the code within this explain is easier to understand what that connection is -- but it is straightforward during init

Reply:
This was too matched the previous arg name + field that init put things in - i.e. to make overall diff smaller. Happy to change if you think we should

Reply:
All good